README.md

```diff
@@ -115,7 +115,7 @@ python chat/base.py
 
 ### Run large models on smaller consumer devices
 
-We support 4-bit quantization (as in QLoRA), LLM.int8, and GPTQ.int4 inference by following [this guide](tutorials/quantize.md).
+We support 4-bit quantization (as in QLoRA), nf4, nf4-dq, fp4, fp4-dq and GPTQ.int4 inference by following [this guide](tutorials/quantize.md).
```

Comment:
Why does LLM.int8 get removed? That's a mode we support and it can be used immediately without any training required, so I would say this is a high value here.

Reply:
you mean [bnb.int8](https://github.com/Lightning-AI/lit-gpt/blob/f13d101a1137503af33ba477db42c16ca5698b2a/generate/base.py#L99), right?
I will add it that - sorry to miss it.

Reply:
Yes, it was previously named LLM.int8 in our lit-llama repo (still think it is a better name than bnb.int8 😅 ). That was the name of the paper and what people know it as :)